modules/creating-customer-seccomp-profile.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * security/seccomp-profiles.adoc
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating seccomp profiles
+You can use the `SeccompProfile` object to create profiles.
+
+`SeccompProfile` objects can restrict system calls (syscalls) within a container, limiting the access of your application.
+
+.Prerequisites
+
+* You have cluster admin permissions.
+* You have created a custom security context constraints (SCC). For more information, see _Additional resources_.
+
+.Procedure
+
+* Create the `SeccompProfile` object:
```

Comment:
The below example should be wrong. The SeccompProfile cr is not available when security-profiles-operator not installed.
$ oc explain SeccompProfile
the server doesn't have a resource type "SeccompProfile"

Reply:
For the non-operator seccompprofile, I used to create a machineconfig to create a custom seccomp file:
```
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: worker
  name: custom-seccomp
spec:
  config:
    ignition:
      version: 2.2.0
    storage:
      files:
      - contents:
          source: data:text/plain;charset=utf-8;base64,ewoJImRlZmF1bHRBY3Rpb24iOiAiU0NNUF9BQ1RfRVJSTk8iLAoJImFyY2hNYXAiOiBbCgkJewoJCQkiYXJjaGl0ZWN0dXJlIjogIlNDTVBfQVJDSF9YODZfNjQiLAoJCQkic3ViQXJjaGl0ZWN0dXJlcyI6IFsKCQkJCSJTQ01QX0FSQ0hfWDg2IiwKCQkJCSJTQ01QX0FSQ0hfWDMyIgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX0FBUkNINjQiLAoJCQkic3ViQXJjaGl0ZWN0dXJlcyI6IFsKCQkJCSJTQ01QX0FSQ0hfQVJNIgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX01JUFM2NCIsCgkJCSJzdWJBcmNoaXRlY3R1cmVzIjogWwoJCQkJIlNDTVBfQVJDSF9NSVBTIiwKCQkJCSJTQ01QX0FSQ0hfTUlQUzY0TjMyIgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX01JUFM2NE4zMiIsCgkJCSJzdWJBcmNoaXRlY3R1cmVzIjogWwoJCQkJIlNDTVBfQVJDSF9NSVBTIiwKCQkJCSJTQ01QX0FSQ0hfTUlQUzY0IgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX01JUFNFTDY0IiwKCQkJInN1YkFyY2hpdGVjdHVyZXMiOiBbCgkJCQkiU0NNUF9BUkNIX01JUFNFTCIsCgkJCQkiU0NNUF9BUkNIX01JUFNFTDY0TjMyIgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX01JUFNFTDY0TjMyIiwKCQkJInN1YkFyY2hpdGVjdHVyZXMiOiBbCgkJCQkiU0NNUF9BUkNIX01JUFNFTCIsCgkJCQkiU0NNUF9BUkNIX01JUFNFTDY0IgoJCQldCgkJfSwKCQl7CgkJCSJhcmNoaXRlY3R1cmUiOiAiU0NNUF9BUkNIX1MzOTBYIiwKCQkJInN1YkFyY2hpdGVjdHVyZXMiOiBbCgkJCQkiU0NNUF9BUkNIX1MzOTAiCgkJCV0KCQl9CgldLAoJInN5c2NhbGxzIjogWwoJCXsKCQkJIm5hbWVzIjogWwoJCQkJIl9sbHNlZWsiLAoJCQkJIl9uZXdzZWxlY3QiLAoJCQkJImFjY2VwdCIsCgkJCQkiYWNjZXB0NCIsCgkJCQkiYWNjZXNzIiwKCQkJCSJhZGp0aW1leCIsCgkJCQkiYWxhcm0iLAoJCQkJImJpbmQiLAoJCQkJImJyayIsCgkJCQkiY2FwZ2V0IiwKCQkJCSJjYXBzZXQiLAoJCQkJImNoZGlyIiwKCQkJCSJjaG1vZCIsCgkJCQkiY2hvd24iLAoJCQkJImNob3duMzIiLAoJCQkJImNsb2NrX2dldHJlcyIsCgkJCQkiY2xvY2tfZ2V0dGltZSIsCgkJCQkiY2xvY2tfbmFub3NsZWVwIiwKCQkJCSJjbG9zZSIsCgkJCQkiY29ubmVjdCIsCgkJCQkiY29weV9maWxlX3JhbmdlIiwKCQkJCSJjcmVhdCIsCgkJCQkiZHVwIiwKCQkJCSJkdXAyIiwKCQkJCSJkdXAzIiwKCQkJCSJlcG9sbF9jcmVhdGUiLAoJCQkJImVwb2xsX2NyZWF0ZTEiLAoJCQkJImVwb2xsX2N0bCIsCgkJCQkiZXBvbGxfY3RsX29sZCIsCgkJCQkiZXBvbGxfcHdhaXQiLAoJCQkJImVwb2xsX3dhaXQiLAoJCQkJImVwb2xsX3dhaXRfb2xkIiwKCQkJCSJldmVudGZkIiwKCQkJCSJldmVudGZkMiIsCgkJCQkiZXhlY3ZlIiwKCQkJCSJleGVjdmVhdCIsCgkJCQkiZXhpdCIsCgkJCQkiZXhpdF9ncm91cCIsCgkJCQkiZmFjY2Vzc2F0IiwKCQkJCSJmYWR2aXNlNjQiLAoJCQkJImZhZHZpc2U2NF82NCIsCgkJCQkiZmFsbG9jYXRlIiwKCQkJCSJmYW5vdGlmeV9tYXJrIiwKCQkJCSJmY2hkaXIiLAoJCQkJImZjaG1vZCIsCgkJCQkiZmNobW9kYXQiLAoJCQkJImZjaG93biIsCgkJCQkiZmNob3duMzIiLAoJCQkJImZjaG93bmF0IiwKCQkJCSJmY250bCIsCgkJCQkiZmNudGw2NCIsCgkJCQkiZmRhdGFzeW5jIiwKCQkJCSJmZ2V0eGF0dHIiLAoJCQkJImZsaXN0eGF0dHIiLAoJCQkJImZsb2NrIiwKCQkJCSJmb3JrIiwKCQkJCSJmcmVtb3ZleGF0dHIiLAoJCQkJImZzZXR4YXR0ciIsCgkJCQkiZnN0YXQiLAoJCQkJImZzdGF0NjQiLAoJCQkJImZzdGF0YXQ2NCIsCgkJCQkiZnN0YXRmcyIsCgkJCQkiZnN0YXRmczY0IiwKCQkJCSJmc3luYyIsCgkJCQkiZnRydW5jYXRlIiwKCQkJCSJmdHJ1bmNhdGU2NCIsCgkJCQkiZnV0ZXgiLAoJCQkJImZ1dGltZXNhdCIsCgkJCQkiZ2V0X3JvYnVzdF9saXN0IiwKCQkJCSJnZXRfdGhyZWFkX2FyZWEiLAoJCQkJImdldGNwdSIsCgkJCQkiZ2V0Y3dkIiwKCQkJCSJnZXRkZW50cyIsCgkJCQkiZ2V0ZGVudHM2NCIsCgkJCQkiZ2V0ZWdpZCIsCgkJCQkiZ2V0ZWdpZDMyIiwKCQkJCSJnZXRldWlkIiwKCQkJCSJnZXRldWlkMzIiLAoJCQkJImdldGdpZCIsCgkJCQkiZ2V0Z2lkMzIiLAoJCQkJImdldGdyb3VwcyIsCgkJCQkiZ2V0Z3JvdXBzMzIiLAoJCQkJImdldGl0aW1lciIsCgkJCQkiZ2V0cGVlcm5hbWUiLAoJCQkJImdldHBnaWQiLAoJCQkJImdldHBncnAiLAoJCQkJImdldHBpZCIsCgkJCQkiZ2V0cHBpZCIsCgkJCQkiZ2V0cHJpb3JpdHkiLAoJCQkJImdldHJhbmRvbSIsCgkJCQkiZ2V0cmVzZ2lkIiwKCQkJCSJnZXRyZXNnaWQzMiIsCgkJCQkiZ2V0cmVzdWlkIiwKCQkJCSJnZXRyZXN1aWQzMiIsCgkJCQkiZ2V0cmxpbWl0IiwKCQkJCSJnZXRydXNhZ2UiLAoJCQkJImdldHNpZCIsCgkJCQkiZ2V0c29ja25hbWUiLAoJCQkJImdldHNvY2tvcHQiLAoJCQkJImdldHRpZCIsCgkJCQkiZ2V0dGltZW9mZGF5IiwKCQkJCSJnZXR1aWQiLAoJCQkJImdldHVpZDMyIiwKCQkJCSJnZXR4YXR0ciIsCgkJCQkiaW5vdGlmeV9hZGRfd2F0Y2giLAoJCQkJImlub3RpZnlfaW5pdCIsCgkJCQkiaW5vdGlmeV9pbml0MSIsCgkJCQkiaW5vdGlmeV9ybV93YXRjaCIsCgkJCQkiaW9fY2FuY2VsIiwKCQkJCSJpb19kZXN0cm95IiwKCQkJCSJpb19nZXRldmVudHMiLAoJCQkJImlvX3NldHVwIiwKCQkJCSJpb19zdWJtaXQiLAoJCQkJImlvY3RsIiwKCQkJCSJpb3ByaW9fZ2V0IiwKCQkJCSJpb3ByaW9fc2V0IiwKCQkJCSJpcGMiLAoJCQkJImtpbGwiLAoJCQkJImxjaG93biIsCgkJCQkibGNob3duMzIiLAoJCQkJImxnZXR4YXR0ciIsCgkJCQkibGluayIsCgkJCQkibGlua2F0IiwKCQkJCSJsaXN0ZW4iLAoJCQkJImxpc3R4YXR0ciIsCgkJCQkibGxpc3R4YXR0ciIsCgkJCQkibHJlbW92ZXhhdHRyIiwKCQkJCSJsc2VlayIsCgkJCQkibHNldHhhdHRyIiwKCQkJCSJsc3RhdCIsCgkJCQkibHN0YXQ2NCIsCgkJCQkibWFkdmlzZSIsCgkJCQkibWVtZmRfY3JlYXRlIiwKCQkJCSJtaW5jb3JlIiwKCQkJCSJta2RpciIsCgkJCQkibWtkaXJhdCIsCgkJCQkibWtub2QiLAoJCQkJIm1rbm9kYXQiLAoJCQkJIm1sb2NrIiwKCQkJCSJtbG9jazIiLAoJCQkJIm1sb2NrYWxsIiwKCQkJCSJtbWFwIiwKCQkJCSJtbWFwMiIsCgkJCQkibW91bnQiLAoJCQkJIm1wcm90ZWN0IiwKCQkJCSJtcV9nZXRzZXRhdHRyIiwKCQkJCSJtcV9ub3RpZnkiLAoJCQkJIm1xX29wZW4iLAoJCQkJIm1xX3RpbWVkcmVjZWl2ZSIsCgkJCQkibXFfdGltZWRzZW5kIiwKCQkJCSJtcV91bmxpbmsiLAoJCQkJIm1yZW1hcCIsCgkJCQkibXNnY3RsIiwKCQkJCSJtc2dnZXQiLAoJCQkJIm1zZ3JjdiIsCgkJCQkibXNnc25kIiwKCQkJCSJtc3luYyIsCgkJCQkibXVubG9jayIsCgkJCQkibXVubG9ja2FsbCIsCgkJCQkibXVubWFwIiwKCQkJCSJuYW1lX3RvX2hhbmRsZV9hdCIsCgkJCQkibmFub3NsZWVwIiwKCQkJCSJuZXdmc3RhdGF0IiwKCQkJCSJvcGVuIiwKCQkJCSJvcGVuYXQiLAoJCQkJInBhdXNlIiwKCQkJCSJwaXBlIiwKCQkJCSJwaXBlMiIsCgkJCQkicG9sbCIsCgkJCQkicHBvbGwiLAoJCQkJInByY3RsIiwKCQkJCSJwcmVhZDY0IiwKCQkJCSJwcmVhZHYiLAoJCQkJInByZWFkdjIiLAoJCQkJInBybGltaXQ2NCIsCgkJCQkicHNlbGVjdDYiLAoJCQkJInB3cml0ZTY0IiwKCQkJCSJwd3JpdGV2IiwKCQkJCSJwd3JpdGV2MiIsCgkJCQkicmVhZCIsCgkJCQkicmVhZGFoZWFkIiwKCQkJCSJyZWFkbGluayIsCgkJCQkicmVhZGxpbmthdCIsCgkJCQkicmVhZHYiLAoJCQkJInJlYm9vdCIsCgkJCQkicmVjdiIsCgkJCQkicmVjdmZyb20iLAoJCQkJInJlY3ZtbXNnIiwKCQkJCSJyZWN2bXNnIiwKCQkJCSJyZW1hcF9maWxlX3BhZ2VzIiwKCQkJCSJyZW1vdmV4YXR0ciIsCgkJCQkicmVuYW1lIiwKCQkJCSJyZW5hbWVhdCIsCgkJCQkicmVuYW1lYXQyIiwKCQkJCSJyZXN0YXJ0X3N5c2NhbGwiLAoJCQkJInJtZGlyIiwKCQkJCSJydF9zaWdhY3Rpb24iLAoJCQkJInJ0X3NpZ3BlbmRpbmciLAoJCQkJInJ0X3NpZ3Byb2NtYXNrIiwKCQkJCSJydF9zaWdxdWV1ZWluZm8iLAoJCQkJInJ0X3NpZ3JldHVybiIsCgkJCQkicnRfc2lnc3VzcGVuZCIsCgkJCQkicnRfc2lndGltZWR3YWl0IiwKCQkJCSJydF90Z3NpZ3F1ZXVlaW5mbyIsCgkJCQkic2NoZWRfZ2V0X3ByaW9yaXR5X21heCIsCgkJCQkic2NoZWRfZ2V0X3ByaW9yaXR5X21pbiIsCgkJCQkic2NoZWRfZ2V0YWZmaW5pdHkiLAoJCQkJInNjaGVkX2dldGF0dHIiLAoJCQkJInNjaGVkX2dldHBhcmFtIiwKCQkJCSJzY2hlZF9nZXRzY2hlZHVsZXIiLAoJCQkJInNjaGVkX3JyX2dldF9pbnRlcnZhbCIsCgkJCQkic2NoZWRfc2V0YWZmaW5pdHkiLAoJCQkJInNjaGVkX3NldGF0dHIiLAoJCQkJInNjaGVkX3NldHBhcmFtIiwKCQkJCSJzY2hlZF9zZXRzY2hlZHVsZXIiLAoJCQkJInNjaGVkX3lpZWxkIiwKCQkJCSJzZWNjb21wIiwKCQkJCSJzZWxlY3QiLAoJCQkJInNlbWN0bCIsCgkJCQkic2VtZ2V0IiwKCQkJCSJzZW1vcCIsCgkJCQkic2VtdGltZWRvcCIsCgkJCQkic2VuZCIsCgkJCQkic2VuZGZpbGUiLAoJCQkJInNlbmRmaWxlNjQiLAoJCQkJInNlbmRtbXNnIiwKCQkJCSJzZW5kbXNnIiwKCQkJCSJzZW5kdG8iLAoJCQkJInNldF9yb2J1c3RfbGlzdCIsCgkJCQkic2V0X3RocmVhZF9hcmVhIiwKCQkJCSJzZXRfdGlkX2FkZHJlc3MiLAoJCQkJInNldGZzZ2lkIiwKCQkJCSJzZXRmc2dpZDMyIiwKCQkJCSJzZXRmc3VpZCIsCgkJCQkic2V0ZnN1aWQzMiIsCgkJCQkic2V0Z2lkIiwKCQkJCSJzZXRnaWQzMiIsCgkJCQkic2V0Z3JvdXBzIiwKCQkJCSJzZXRncm91cHMzMiIsCgkJCQkic2V0aXRpbWVyIiwKCQkJCSJzZXRwZ2lkIiwKCQkJCSJzZXRwcmlvcml0eSIsCgkJCQkic2V0cmVnaWQiLAoJCQkJInNldHJlZ2lkMzIiLAoJCQkJInNldHJlc2dpZCIsCgkJCQkic2V0cmVzZ2lkMzIiLAoJCQkJInNldHJlc3VpZCIsCgkJCQkic2V0cmVzdWlkMzIiLAoJCQkJInNldHJldWlkIiwKCQkJCSJzZXRyZXVpZDMyIiwKCQkJCSJzZXRybGltaXQiLAoJCQkJInNldHNpZCIsCgkJCQkic2V0c29ja29wdCIsCgkJCQkic2V0dWlkIiwKCQkJCSJzZXR1aWQzMiIsCgkJCQkic2V0eGF0dHIiLAoJCQkJInNobWF0IiwKCQkJCSJzaG1jdGwiLAoJCQkJInNobWR0IiwKCQkJCSJzaG1nZXQiLAoJCQkJInNodXRkb3duIiwKCQkJCSJzaWdhbHRzdGFjayIsCgkJCQkic2lnbmFsZmQiLAoJCQkJInNpZ25hbGZkNCIsCgkJCQkic2lncmV0dXJuIiwKCQkJCSJzb2NrZXQiLAoJCQkJInNvY2tldGNhbGwiLAoJCQkJInNvY2tldHBhaXIiLAoJCQkJInNwbGljZSIsCgkJCQkic3RhdGZzIiwKCQkJCSJzdGF0ZnM2NCIsCgkJCQkic3RhdHgiLAoJCQkJInN5bWxpbmsiLAoJCQkJInN5bWxpbmthdCIsCgkJCQkic3luYyIsCgkJCQkic3luY19maWxlX3JhbmdlIiwKCQkJCSJzeW5jZnMiLAoJCQkJInN5c2luZm8iLAoJCQkJInN5c2xvZyIsCgkJCQkidGVlIiwKCQkJCSJ0Z2tpbGwiLAoJCQkJInRpbWUiLAoJCQkJInRpbWVyX2NyZWF0ZSIsCgkJCQkidGltZXJfZGVsZXRlIiwKCQkJCSJ0aW1lcl9nZXRvdmVycnVuIiwKCQkJCSJ0aW1lcl9nZXR0aW1lIiwKCQkJCSJ0aW1lcl9zZXR0aW1lIiwKCQkJCSJ0aW1lcmZkX2NyZWF0ZSIsCgkJCQkidGltZXJmZF9nZXR0aW1lIiwKCQkJCSJ0aW1lcmZkX3NldHRpbWUiLAoJCQkJInRpbWVzIiwKCQkJCSJ0a2lsbCIsCgkJCQkidHJ1bmNhdGUiLAoJCQkJInRydW5jYXRlNjQiLAoJCQkJInVnZXRybGltaXQiLAoJCQkJInVtYXNrIiwKCQkJCSJ1bW91bnQiLAoJCQkJInVtb3VudDIiLAoJCQkJInVuYW1lIiwKCQkJCSJ1bmxpbmsiLAoJCQkJInVubGlua2F0IiwKCQkJCSJ1bnNoYXJlIiwKCQkJCSJ1dGltZSIsCgkJCQkidXRpbWVuc2F0IiwKCQkJCSJ1dGltZXMiLAoJCQkJInZmb3JrIiwKCQkJCSJ2bXNwbGljZSIsCgkJCQkid2FpdDQiLAoJCQkJIndhaXRpZCIsCgkJCQkid2FpdHBpZCIsCgkJCQkid3JpdGUiLAoJCQkJIndyaXRldiIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHt9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkicGVyc29uYWxpdHkiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFsKCQkJCXsKCQkJCQkiaW5kZXgiOiAwLAoJCQkJCSJ2YWx1ZSI6IDAsCgkJCQkJInZhbHVlVHdvIjogMCwKCQkJCQkib3AiOiAiU0NNUF9DTVBfRVEiCgkJCQl9CgkJCV0sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHt9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkicGVyc29uYWxpdHkiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFsKCQkJCXsKCQkJCQkiaW5kZXgiOiAwLAoJCQkJCSJ2YWx1ZSI6IDgsCgkJCQkJInZhbHVlVHdvIjogMCwKCQkJCQkib3AiOiAiU0NNUF9DTVBfRVEiCgkJCQl9CgkJCV0sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHt9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkicGVyc29uYWxpdHkiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFsKCQkJCXsKCQkJCQkiaW5kZXgiOiAwLAoJCQkJCSJ2YWx1ZSI6IDEzMTA3MiwKCQkJCQkidmFsdWVUd28iOiAwLAoJCQkJCSJvcCI6ICJTQ01QX0NNUF9FUSIKCQkJCX0KCQkJXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjoge30sCgkJCSJleGNsdWRlcyI6IHt9CgkJfSwKCQl7CgkJCSJuYW1lcyI6IFsKCQkJCSJwZXJzb25hbGl0eSIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogWwoJCQkJewoJCQkJCSJpbmRleCI6IDAsCgkJCQkJInZhbHVlIjogMTMxMDgwLAoJCQkJCSJ2YWx1ZVR3byI6IDAsCgkJCQkJIm9wIjogIlNDTVBfQ01QX0VRIgoJCQkJfQoJCQldLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7fSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJInBlcnNvbmFsaXR5IgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbCgkJCQl7CgkJCQkJImluZGV4IjogMCwKCQkJCQkidmFsdWUiOiA0Mjk0OTY3Mjk1LAoJCQkJCSJ2YWx1ZVR3byI6IDAsCgkJCQkJIm9wIjogIlNDTVBfQ01QX0VRIgoJCQkJfQoJCQldLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7fSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJInN5bmNfZmlsZV9yYW5nZTIiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFtdLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7CgkJCQkiYXJjaGVzIjogWwoJCQkJCSJwcGM2NGxlIgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiYXJtX2ZhZHZpc2U2NF82NCIsCgkJCQkiYXJtX3N5bmNfZmlsZV9yYW5nZSIsCgkJCQkic3luY19maWxlX3JhbmdlMiIsCgkJCQkiYnJlYWtwb2ludCIsCgkJCQkiY2FjaGVmbHVzaCIsCgkJCQkic2V0X3RscyIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJhcmNoZXMiOiBbCgkJCQkJImFybSIsCgkJCQkJImFybTY0IgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiYXJjaF9wcmN0bCIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJhcmNoZXMiOiBbCgkJCQkJImFtZDY0IiwKCQkJCQkieDMyIgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkibW9kaWZ5X2xkdCIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJhcmNoZXMiOiBbCgkJCQkJImFtZDY0IiwKCQkJCQkieDMyIiwKCQkJCQkieDg2IgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiczM5MF9wY2lfbW1pb19yZWFkIiwKCQkJCSJzMzkwX3BjaV9tbWlvX3dyaXRlIiwKCQkJCSJzMzkwX3J1bnRpbWVfaW5zdHIiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFtdLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7CgkJCQkiYXJjaGVzIjogWwoJCQkJCSJzMzkwIiwKCQkJCQkiczM5MHgiCgkJCQldCgkJCX0sCgkJCSJleGNsdWRlcyI6IHt9CgkJfSwKCQl7CgkJCSJuYW1lcyI6IFsKCQkJCSJvcGVuX2J5X2hhbmRsZV9hdCIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJjYXBzIjogWwoJCQkJCSJDQVBfREFDX1JFQURfU0VBUkNIIgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiYnBmIiwKCQkJCSJjbG9uZSIsCgkJCQkiZmFub3RpZnlfaW5pdCIsCgkJCQkibG9va3VwX2Rjb29raWUiLAoJCQkJIm1vdW50IiwKCQkJCSJuYW1lX3RvX2hhbmRsZV9hdCIsCgkJCQkicGVyZl9ldmVudF9vcGVuIiwKCQkJCSJxdW90YWN0bCIsCgkJCQkic2V0ZG9tYWlubmFtZSIsCgkJCQkic2V0aG9zdG5hbWUiLAoJCQkJInNldG5zIiwKCQkJCSJ1bW91bnQiLAoJCQkJInVtb3VudDIiLAoJCQkJInVuc2hhcmUiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFtdLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7CgkJCQkiY2FwcyI6IFsKCQkJCQkiQ0FQX1NZU19BRE1JTiIKCQkJCV0KCQkJfSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJImNsb25lIgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbCgkJCQl7CgkJCQkJImluZGV4IjogMCwKCQkJCQkidmFsdWUiOiAyMDgwNTA1ODU2LAoJCQkJCSJ2YWx1ZVR3byI6IDAsCgkJCQkJIm9wIjogIlNDTVBfQ01QX01BU0tFRF9FUSIKCQkJCX0KCQkJXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjoge30sCgkJCSJleGNsdWRlcyI6IHsKCQkJCSJjYXBzIjogWwoJCQkJCSJDQVBfU1lTX0FETUlOIgoJCQkJXSwKCQkJCSJhcmNoZXMiOiBbCgkJCQkJInMzOTAiLAoJCQkJCSJzMzkweCIKCQkJCV0KCQkJfQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiY2xvbmUiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFsKCQkJCXsKCQkJCQkiaW5kZXgiOiAxLAoJCQkJCSJ2YWx1ZSI6IDIwODA1MDU4NTYsCgkJCQkJInZhbHVlVHdvIjogMCwKCQkJCQkib3AiOiAiU0NNUF9DTVBfTUFTS0VEX0VRIgoJCQkJfQoJCQldLAoJCQkiY29tbWVudCI6ICJzMzkwIHBhcmFtZXRlciBvcmRlcmluZyBmb3IgY2xvbmUgaXMgZGlmZmVyZW50IiwKCQkJImluY2x1ZGVzIjogewoJCQkJImFyY2hlcyI6IFsKCQkJCQkiczM5MCIsCgkJCQkJInMzOTB4IgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7CgkJCQkiY2FwcyI6IFsKCQkJCQkiQ0FQX1NZU19BRE1JTiIKCQkJCV0KCQkJfQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkicmVib290IgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjogewoJCQkJImNhcHMiOiBbCgkJCQkJIkNBUF9TWVNfQk9PVCIKCQkJCV0KCQkJfSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJImNocm9vdCIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJjYXBzIjogWwoJCQkJCSJDQVBfU1lTX0NIUk9PVCIKCQkJCV0KCQkJfSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJImRlbGV0ZV9tb2R1bGUiLAoJCQkJImluaXRfbW9kdWxlIiwKCQkJCSJmaW5pdF9tb2R1bGUiLAoJCQkJInF1ZXJ5X21vZHVsZSIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJjYXBzIjogWwoJCQkJCSJDQVBfU1lTX01PRFVMRSIKCQkJCV0KCQkJfSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJImdldF9tZW1wb2xpY3kiLAoJCQkJIm1iaW5kIiwKCQkJCSJuYW1lX3RvX2hhbmRsZV9hdCIsCgkJCQkic2V0X21lbXBvbGljeSIKCQkJXSwKCQkJImFjdGlvbiI6ICJTQ01QX0FDVF9BTExPVyIsCgkJCSJhcmdzIjogW10sCgkJCSJjb21tZW50IjogIiIsCgkJCSJpbmNsdWRlcyI6IHsKCQkJCSJjYXBzIjogWwoJCQkJCSJDQVBfU1lTX05JQ0UiCgkJCQldCgkJCX0sCgkJCSJleGNsdWRlcyI6IHt9CgkJfSwKCQl7CgkJCSJuYW1lcyI6IFsKCQkJCSJhY2N0IgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjogewoJCQkJImNhcHMiOiBbCgkJCQkJIkNBUF9TWVNfUEFDQ1QiCgkJCQldCgkJCX0sCgkJCSJleGNsdWRlcyI6IHt9CgkJfSwKCQl7CgkJCSJuYW1lcyI6IFsKCQkJCSJrY21wIiwKCQkJCSJwcm9jZXNzX3ZtX3JlYWR2IiwKCQkJCSJwcm9jZXNzX3ZtX3dyaXRldiIsCgkJCQkicHRyYWNlIgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjogewoJCQkJImNhcHMiOiBbCgkJCQkJIkNBUF9TWVNfUFRSQUNFIgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0sCgkJewoJCQkibmFtZXMiOiBbCgkJCQkiaW9wbCIsCgkJCQkiaW9wZXJtIgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjogewoJCQkJImNhcHMiOiBbCgkJCQkJIkNBUF9TWVNfUkFXSU8iCgkJCQldCgkJCX0sCgkJCSJleGNsdWRlcyI6IHt9CgkJfSwKCQl7CgkJCSJuYW1lcyI6IFsKCQkJCSJzZXR0aW1lb2ZkYXkiLAoJCQkJInN0aW1lIiwKCQkJCSJjbG9ja19zZXR0aW1lIgoJCQldLAoJCQkiYWN0aW9uIjogIlNDTVBfQUNUX0FMTE9XIiwKCQkJImFyZ3MiOiBbXSwKCQkJImNvbW1lbnQiOiAiIiwKCQkJImluY2x1ZGVzIjogewoJCQkJImNhcHMiOiBbCgkJCQkJIkNBUF9TWVNfVElNRSIKCQkJCV0KCQkJfSwKCQkJImV4Y2x1ZGVzIjoge30KCQl9LAoJCXsKCQkJIm5hbWVzIjogWwoJCQkJInZoYW5ndXAiCgkJCV0sCgkJCSJhY3Rpb24iOiAiU0NNUF9BQ1RfQUxMT1ciLAoJCQkiYXJncyI6IFtdLAoJCQkiY29tbWVudCI6ICIiLAoJCQkiaW5jbHVkZXMiOiB7CgkJCQkiY2FwcyI6IFsKCQkJCQkiQ0FQX1NZU19UVFlfQ09ORklHIgoJCQkJXQoJCQl9LAoJCQkiZXhjbHVkZXMiOiB7fQoJCX0KCV0KfQo=
        filesystem: root
        mode: 0644
        path: /var/lib/kubelet/seccomp/seccomp-nostat.json
```
After decode, the original json file is:
```
{
	"defaultAction": "SCMP_ACT_ERRNO",
	"archMap": [
		{
			"architecture": "SCMP_ARCH_X86_64",
			"subArchitectures": [
				"SCMP_ARCH_X86",
				"SCMP_ARCH_X32"
			]
		},
		{
			"architecture": "SCMP_ARCH_AARCH64",
			"subArchitectures": [
				"SCMP_ARCH_ARM"
			]
		},
		{
			"architecture": "SCMP_ARCH_MIPS64",
			"subArchitectures": [
				"SCMP_ARCH_MIPS",
				"SCMP_ARCH_MIPS64N32"
			]
		},
		{
			"architecture": "SCMP_ARCH_MIPS64N32",
			"subArchitectures": [
				"SCMP_ARCH_MIPS",
				"SCMP_ARCH_MIPS64"
			]
		},
		{
			"architecture": "SCMP_ARCH_MIPSEL64",
			"subArchitectures": [
				"SCMP_ARCH_MIPSEL",
				"SCMP_ARCH_MIPSEL64N32"
			]
		},
		{
			"architecture": "SCMP_ARCH_MIPSEL64N32",
			"subArchitectures": [
				"SCMP_ARCH_MIPSEL",
				"SCMP_ARCH_MIPSEL64"
			]
		},
		{
			"architecture": "SCMP_ARCH_S390X",
			"subArchitectures": [
				"SCMP_ARCH_S390"
			]
		}
	],
	"syscalls": [
		{
			"names": [
				"_llseek",
				"_newselect",
				"accept",
				"accept4",
				"access",
				"adjtimex",
				"alarm",
				"bind",
				"brk",
				"capget",
				"capset",
				"chdir",
				"chmod",
				"chown",
				"chown32",
				"clock_getres",
				"clock_gettime",
				"clock_nanosleep",
				"close",
				"connect",
				"copy_file_range",
				"creat",
				"dup",
				"dup2",
				"dup3",
				"epoll_create",
				"epoll_create1",
				"epoll_ctl",
				"epoll_ctl_old",
				"epoll_pwait",
				"epoll_wait",
				"epoll_wait_old",
				"eventfd",
				"eventfd2",
				"execve",
				"execveat",
				"exit",
				"exit_group",
				"faccessat",
				"fadvise64",
				"fadvise64_64",
				"fallocate",
				"fanotify_mark",
				"fchdir",
				"fchmod",
				"fchmodat",
				"fchown",
				"fchown32",
				"fchownat",
				"fcntl",
				"fcntl64",
				"fdatasync",
				"fgetxattr",
				"flistxattr",
				"flock",
				"fork",
				"fremovexattr",
				"fsetxattr",
				"fstat",
				"fstat64",
				"fstatat64",
				"fstatfs",
				"fstatfs64",
				"fsync",
				"ftruncate",
				"ftruncate64",
				"futex",
				"futimesat",
				"get_robust_list",
				"get_thread_area",
				"getcpu",
				"getcwd",
				"getdents",
				"getdents64",
				"getegid",
				"getegid32",
				"geteuid",
				"geteuid32",
				"getgid",
				"getgid32",
				"getgroups",
				"getgroups32",
				"getitimer",
				"getpeername",
				"getpgid",
				"getpgrp",
				"getpid",
				"getppid",
				"getpriority",
				"getrandom",
				"getresgid",
				"getresgid32",
				"getresuid",
				"getresuid32",
				"getrlimit",
				"getrusage",
				"getsid",
				"getsockname",
				"getsockopt",
				"gettid",
				"gettimeofday",
				"getuid",
				"getuid32",
				"getxattr",
				"inotify_add_watch",
				"inotify_init",
				"inotify_init1",
				"inotify_rm_watch",
				"io_cancel",
				"io_destroy",
				"io_getevents",
				"io_setup",
				"io_submit",
				"ioctl",
				"ioprio_get",
				"ioprio_set",
				"ipc",
				"kill",
				"lchown",
				"lchown32",
				"lgetxattr",
				"link",
				"linkat",
				"listen",
				"listxattr",
				"llistxattr",
				"lremovexattr",
				"lseek",
				"lsetxattr",
				"lstat",
				"lstat64",
				"madvise",
				"memfd_create",
				"mincore",
				"mkdir",
				"mkdirat",
				"mknod",
				"mknodat",
				"mlock",
				"mlock2",
				"mlockall",
				"mmap",
				"mmap2",
				"mount",
				"mprotect",
				"mq_getsetattr",
				"mq_notify",
				"mq_open",
				"mq_timedreceive",
				"mq_timedsend",
				"mq_unlink",
				"mremap",
				"msgctl",
				"msgget",
				"msgrcv",
				"msgsnd",
				"msync",
				"munlock",
				"munlockall",
				"munmap",
				"name_to_handle_at",
				"nanosleep",
				"newfstatat",
				"open",
				"openat",
				"pause",
				"pipe",
				"pipe2",
				"poll",
				"ppoll",
				"prctl",
				"pread64",
				"preadv",
				"preadv2",
				"prlimit64",
				"pselect6",
				"pwrite64",
				"pwritev",
				"pwritev2",
				"read",
				"readahead",
				"readlink",
				"readlinkat",
				"readv",
				"reboot",
				"recv",
				"recvfrom",
				"recvmmsg",
				"recvmsg",
				"remap_file_pages",
				"removexattr",
				"rename",
				"renameat",
				"renameat2",
				"restart_syscall",
				"rmdir",
				"rt_sigaction",
				"rt_sigpending",
				"rt_sigprocmask",
				"rt_sigqueueinfo",
				"rt_sigreturn",
				"rt_sigsuspend",
				"rt_sigtimedwait",
				"rt_tgsigqueueinfo",
				"sched_get_priority_max",
				"sched_get_priority_min",
				"sched_getaffinity",
				"sched_getattr",
				"sched_getparam",
				"sched_getscheduler",
				"sched_rr_get_interval",
				"sched_setaffinity",
				"sched_setattr",
				"sched_setparam",
				"sched_setscheduler",
				"sched_yield",
				"seccomp",
				"select",
				"semctl",
				"semget",
				"semop",
				"semtimedop",
				"send",
				"sendfile",
				"sendfile64",
				"sendmmsg",
				"sendmsg",
				"sendto",
				"set_robust_list",
				"set_thread_area",
				"set_tid_address",
				"setfsgid",
				"setfsgid32",
				"setfsuid",
				"setfsuid32",
				"setgid",
				"setgid32",
				"setgroups",
				"setgroups32",
				"setitimer",
				"setpgid",
				"setpriority",
				"setregid",
				"setregid32",
				"setresgid",
				"setresgid32",
				"setresuid",
				"setresuid32",
				"setreuid",
				"setreuid32",
				"setrlimit",
				"setsid",
				"setsockopt",
				"setuid",
				"setuid32",
				"setxattr",
				"shmat",
				"shmctl",
				"shmdt",
				"shmget",
				"shutdown",
				"sigaltstack",
				"signalfd",
				"signalfd4",
				"sigreturn",
				"socket",
				"socketcall",
				"socketpair",
				"splice",
				"statfs",
				"statfs64",
				"statx",
				"symlink",
				"symlinkat",
				"sync",
				"sync_file_range",
				"syncfs",
				"sysinfo",
				"syslog",
				"tee",
				"tgkill",
				"time",
				"timer_create",
				"timer_delete",
				"timer_getoverrun",
				"timer_gettime",
				"timer_settime",
				"timerfd_create",
				"timerfd_gettime",
				"timerfd_settime",
				"times",
				"tkill",
				"truncate",
				"truncate64",
				"ugetrlimit",
				"umask",
				"umount",
				"umount2",
				"uname",
				"unlink",
				"unlinkat",
				"unshare",
				"utime",
				"utimensat",
				"utimes",
				"vfork",
				"vmsplice",
				"wait4",
				"waitid",
				"waitpid",
				"write",
				"writev"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"personality"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 0,
					"valueTwo": 0,
					"op": "SCMP_CMP_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"personality"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 8,
					"valueTwo": 0,
					"op": "SCMP_CMP_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"personality"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 131072,
					"valueTwo": 0,
					"op": "SCMP_CMP_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"personality"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 131080,
					"valueTwo": 0,
					"op": "SCMP_CMP_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"personality"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 4294967295,
					"valueTwo": 0,
					"op": "SCMP_CMP_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {}
		},
		{
			"names": [
				"sync_file_range2"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"arches": [
					"ppc64le"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"arm_fadvise64_64",
				"arm_sync_file_range",
				"sync_file_range2",
				"breakpoint",
				"cacheflush",
				"set_tls"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"arches": [
					"arm",
					"arm64"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"arch_prctl"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"arches": [
					"amd64",
					"x32"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"modify_ldt"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"arches": [
					"amd64",
					"x32",
					"x86"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"s390_pci_mmio_read",
				"s390_pci_mmio_write",
				"s390_runtime_instr"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"arches": [
					"s390",
					"s390x"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"open_by_handle_at"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_DAC_READ_SEARCH"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"bpf",
				"clone",
				"fanotify_init",
				"lookup_dcookie",
				"mount",
				"name_to_handle_at",
				"perf_event_open",
				"quotactl",
				"setdomainname",
				"sethostname",
				"setns",
				"umount",
				"umount2",
				"unshare"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_ADMIN"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"clone"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 0,
					"value": 2080505856,
					"valueTwo": 0,
					"op": "SCMP_CMP_MASKED_EQ"
				}
			],
			"comment": "",
			"includes": {},
			"excludes": {
				"caps": [
					"CAP_SYS_ADMIN"
				],
				"arches": [
					"s390",
					"s390x"
				]
			}
		},
		{
			"names": [
				"clone"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [
				{
					"index": 1,
					"value": 2080505856,
					"valueTwo": 0,
					"op": "SCMP_CMP_MASKED_EQ"
				}
			],
			"comment": "s390 parameter ordering for clone is different",
			"includes": {
				"arches": [
					"s390",
					"s390x"
				]
			},
			"excludes": {
				"caps": [
					"CAP_SYS_ADMIN"
				]
			}
		},
		{
			"names": [
				"reboot"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_BOOT"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"chroot"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_CHROOT"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"delete_module",
				"init_module",
				"finit_module",
				"query_module"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_MODULE"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"get_mempolicy",
				"mbind",
				"name_to_handle_at",
				"set_mempolicy"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_NICE"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"acct"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_PACCT"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"kcmp",
				"process_vm_readv",
				"process_vm_writev",
				"ptrace"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_PTRACE"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"iopl",
				"ioperm"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_RAWIO"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"settimeofday",
				"stime",
				"clock_settime"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_TIME"
				]
			},
			"excludes": {}
		},
		{
			"names": [
				"vhangup"
			],
			"action": "SCMP_ACT_ALLOW",
			"args": [],
			"comment": "",
			"includes": {
				"caps": [
					"CAP_SYS_TTY_CONFIG"
				]
			},
			"excludes": {}
		}
	]
}
```